code/player/PlayerCorpse.cs

```diff
@@ -137,11 +138,13 @@ public ConfirmationData GetConfirmationData()
 
         public float HintDistance => 80f;
 
+        public string TextOnTick => TTTLanguage.ActiveLanguage.GetFormattedTranslation(IsIdentified ? "CORPSE_INSPECT" : "CORPSE_IDENTIFY", new object[] { Input.GetKeyWithBinding("+iv_use").ToUpper() });
```

Comment:
Does this even update in the Tick method? If not, then this won't work and we should use the other approach. If it does work, then the corresponding function is executed every tick which means there will be a lookup every tick for every hint which I don't wanna like, not just performance-based, but also coding-wise as this does not offer the possibility to alternate the text on the screen, e.g. by another add-on later.